egs/hub4_spanish/s5/local/chain/run_ivector_common_pitch.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+# This script is called from local/nnet3/run_tdnn.sh and local/chain/run_tdnn.sh (and may eventually
+# be called by more scripts).  It contains the common feature preparation and iVector-related parts
+# of the script.  See those scripts for examples of usage.
+
+
+stage=0
+nj=30
+min_seg_len=1.55  # min length in seconds... we do this because chain training
```

Comment:
This script is kind of out of date.  min_seg_len is something we don't use any more, since we use a range of chunk sizes in the chain model training, so losing small segments is less of an issue.
I suspect there are other issues too.  Better to copy and modify the run_ivector_common.sh of a setup that *is* fully up-to-date.
As far as I can see, the only ones that are fully up-to-date in the sense that they use PCA instead of LDA
for the i-vector UBM (@david-ryan-snyder , we need to update more of them...) are mini_librispeech and ami/s5b, of which mini_librispeech is the better starting point because it doesn't have the microphone issues.
Of course you might want to check whether the pitch matters first, because the mini_librispeech setup
doesn't have that.  [and see if we used a smaller num-gauss for the mini_librispeech because it has
so little data.]

Naturally for the chain training script, you should get one that's up-to-date too.  I think the
mini-librispeech one should be up-to-date, but it doesn't have a TDNN+LSTM system which
 is usually the best.  You can look at WSJ for that system.

I don't like adding new setups that are not using all of the current best practices.  Old setups
are grandfathered in, but for new ones, we should use up-to-date scripts.